superset/views/core.py

```diff
@@ -925,3 +925,13 @@ def welcome(self) -> FlaskResponse:
     @deprecated(new_target="/sqllab/history")
     def sqllab_history(self) -> FlaskResponse:
         return redirect("/sqllab/history")
+
+    @has_access
+    @api
+    @expose("/loader/", methods=("GET",))
```

Comment:
I think everything loader-related will collide with the Theming branch, let's keep the PR focussed on the initial goal